CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-10-01
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
```

Comment:
I have noticed there has been no GA announcement for VPA in the AKS release notes. Maybe we could get it added to this release note as it seems to have gone to GA 27th September. https://azure.microsoft.com/en-gb/updates/ga-vertical-pod-autoscaling-addon-for-aks/

Reply:
Thanks for adding. I just noticed we need to remove the en-us from the URL.

Reply:
Thanks for adding. I just noticed we need to remove the en-us from the URL.

Reply:
Also check the other URLs as the new ones in this pr have en-us also and should be removed.